cupyx/runtime.py

```diff
@@ -23,6 +25,43 @@ def _eval_or_error(func, errors):
         return repr(e)
 
 
+class _InstallInfo(object):
+
+    def __init__(self):
+        cupy_package_root = self._get_cupy_package_root()
```

Comment:
Just a suggestion, I think we can use pip to find the package_root  and wether the package was installed using pip itself.

from pip._internal.utils.misc import get_installed_distributions
filter(lambda x: 'cupy' in x.project_name, get_installed_distributions())

The returned object has a 'location' attribute to get the install path